code/create/create-dta.sh

```diff
@@ -0,0 +1,46 @@
+# First argument: Path to install the DTA_study data
+# Second argument: which sourcedata to use
+# Example-call for installing DTA_sourcedata_reduced:
+# >> ./create_BIDS_DTA.sh . /data/BnB_TEMP/Kadelka/DTA_sourcedata_reduced/
+
+# create a dataset
+datalad create -c hirni $1
+cd $1
+
+# installing the necessary code into the dataset
+datalad install -d . -s git@github.com:TobiasKadelka/build_dta.git code/build_dta
+
+# modify the rules for the spec, so the dta-rules will be used
+git config -f .datalad/config --add datalad.hirni.dicom2spec.rules code/build_dta/code/create/dta_dicom2spec_rules.py
```

Comment:
git-config modifies .datalad/config here. So you want to call datalad-save afterwards.